build.gradle

```diff
@@ -50,7 +50,7 @@ repositories {
 esplugin {
     name project.properties['pluginName']
     description 'Safely expose Elasticsearch REST API'
-    classname 'org.elasticsearch.plugin.readonlyrest.wiring.ReadonlyRestPlugin'
+    classname 'org.elasticsearch.plugin.readonlyrest.es53x.ReadonlyRestPlugin'
```

Comment:
I've changed it to `es`. Next step (I think in next PR) will be moving code from this package to `es53x` module. And there will be also `es52x` module, etc. Each module will have own `build.gradle` file. Then `bin/build.sh` script will have `./gradle assembly [...]` for each module.